tiled/catalog/adapter.py

```diff
@@ -975,6 +976,11 @@ def contains(query, tree):
     return tree.new_variation(conditions=tree.conditions + [condition])
 
 
+def fullText(query, tree):
```

Comment:
[PEP 8](https://peps.python.org/pep-0008/#function-and-variable-names):

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.

> Variable names follow the same convention as function names.

> mixedCase is allowed only in contexts where that’s already the prevailing style (e.g. threading.py), to retain backwards compatibility.

Reply:
Added in [b713504fddbb7f0c064e27900dbcc97473365ceb](https://github.com/bluesky/tiled/pull/628/commits/b713504fddbb7f0c064e27900dbcc97473365ceb)